pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1359,8 +1360,9 @@ func configToStatus(ctx *domainContext, config types.DomainConfig,
 		xv := "xvd" + string(int('a')+i)
 		ds.Vdev = xv
 
+		log.Infof("getting image file location IsContainer(%v), ContainerImageId(%s), ImageSha256(%s)", status.IsContainer, ds.ImageID.String(), dc.ImageSha256)
 		target, err := utils.VerifiedImageFileLocation(status.IsContainer,
-			status.ContainerImageID, dc.ImageSha256)
+			ds.ImageID.String(), dc.ImageSha256)
```

Comment:
Is the intent we shouldn't use ContainerImageId any more?

Reply:
It was blank. Actually depending on it caused errors. As I wrote above, some of the creation of the paths confused me quite a bit (my note for the last thing to clean up was, "check paths for files multiple times"; I left it for last because it was the hardest to get my head around, when it creates paths from what). This way it worked.

Personally, I would like to see all of it extracted to some other utility, but enough for one PR. :-)

Reply:
Actually Kalyan's plan was to use the image uuid for the naming so it would be uniform for all images. But that is TBD.

Reply:
That would be fine. For now, it looks like we use the sha hash for VMs and UUID for containers (probably since there is no hash available for most containers).